channels/serializers.py

```diff
@@ -171,11 +173,11 @@ def get_author_name(self, instance):
         return "[deleted]"
 
     def get_profile_image(self, instance):
-        """Find the Profile for the comment author"""
+        """Find the profile image for the post author"""
         user = self._get_user(instance)
         if user and user.profile.image_small:
             return user.profile.image_small
-        return default_profile_image
+        return urljoin(SITE_BASE_URL, default_profile_image)
```

Comment:
Could you actually make this `settings.SITE_BASE_URL`? It avoids any potential issues with the value at import-time.